### DISCUSSION TITLE
Giving out answers

### DISCUSSION THREAD

**Discussion by 3kh0** [Schweinepriester/github-profile-achievements#372] [Q&A] [answered]
First one to comment gets marked as an answer, hoping to start a trend here. If you comment, please help out others by creating a new discussion so others can get answers!

**Comment by deadnews** [comment, marked as answer]
Done.

**Comment by Snocember** [comment]
👀

**Reply by tanish36** [reply]
Done

**Comment by PatrickDSiqueira** [comment]
I still haven't figured out how to get the Galaxy Brain
How does this markup of accepted answers accepted???
Could anyone help me?

**Reply by 3kh0** [reply]
you get it by getting your answer marked

**Reply by PatrickDSiqueira** [reply]
Show

**Reply by PatrickDSiqueira** [reply]
Dude, could you help me? based on that only one is missing for me

**Reply by 3kh0** [reply]
you only have 2... answer discussions
